content/en/guides/features/rendering-modes.md

```diff
@@ -24,18 +24,24 @@ export default {
 ```
 
 <base-alert type="info">
+
 You do not need to add this to your nuxt config in order for universal mode to be applied as the default mode is universal.
+
 </base-alert>
 
 ## SPA
 
-`mode: 'spa'`: No server-side rendering, only client-side rendering.
+<base-alert type="warning">
+
+Deprecated: please use `ssr: false` instead of `mode: spa`
```

Comment:
Is it necessary to warn here? It makes SPA looking deprecated itself while only configuration option is deprecated.

Reply:
your right. we should remove it from here.